src/components/Tab/Tab.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { BodyRegular, BodyBold } from "../../styles/typography";
+import { StyledTab } from "./styles";
+import { CSSProperties } from "styled-components";
+
+interface TabProps {
+  text: string;
+  img?: string;
+  active?: boolean;
+  onClick?: (active: boolean) => void;
+  style?: CSSProperties;
+}
+
+const Tab = ({ text = "list", active = false, onClick, img, style }: TabProps) => {
+  const [isActive, setIsActive] = useState(active);
+
+  const handleClick = () => {
+    setIsActive(!isActive);
+    if (onClick) onClick(!isActive);
```

Comment:
you can change it to:
```suggestion
     onClick?.(!isActive);
```

Reply:
Made change.